.buildkite/pipeline.yml

```diff
@@ -0,0 +1,51 @@
+# Nodes with values to reuse in the pipeline.
+common_params:
+  plugins: &common_plugins
+  - &bash_cache automattic/bash-cache#2.10.0: ~
```

Comment:
Nitpick. This doesn't hurt, but doesn't do anything useful either. In [the docs](https://buildkite.com/docs/plugins/using#configuring-plugins) this is the syntax used for "no configuration":

![image](https://user-images.githubusercontent.com/1218433/203533463-fe783e31-e877-423a-94f3-f5483ac5240d.png)


```suggestion
  - &bash_cache automattic/bash-cache#2.10.0
```

Reply:
TIL. I copied this file from another project 😄